docs/protocol/terminology.mdx

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 2
+title: "Terminology"
```

Comment:
I am not sure whether I understand the difference between "Terminology" and "Glossary" pages. Both explain protocol concepts, so it confuses me.

Reply:
I like it! @dimast-x , @philanton ?